CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added `on_bcp_error` and `bcp_error_log_path` parameters in `BCPTask`
 - Added ability to process queries which result exceed SAP's character per low limit in `SAPRFC` source
 ### Changed
+- Cahnged `CheckColumnOrder` and `ADLSToAzureSQL` to handle appending to non existing table
```

Comment:
- typo in "Changed"
- I would add "task" and "flow" words to be a bit more explicit, so "Changed `CheckColumnOrder` task and `ADLSToAzureSQL` flow ..."